include/ddc/kernels/splines/spline_builder.hpp

```diff
@@ -57,34 +81,52 @@ class SplineBuilder
     using tag_type = typename InterpolationMesh::continuous_dimension_type;
 
 public:
+    /**
+     * @brief The type of the Kokkos execution space used by this class.
+     */
     using exec_space = ExecSpace;
 
+    /**
+     * @brief The type of the Kokkos memory space used by this class.
+     */
     using memory_space = MemorySpace;
 
     /**
-     * @brief The type of the interpolation mesh used by this class.
+     * @brief The type of the interpolation mesh (discrete dimension of interest) used by this class.
      */
     using interpolation_mesh_type = InterpolationMesh;
 
     /**
-     * @brief The type of the BSplines which are compatible with this class.
+     * @brief The BSplines dimension.
      */
     using bsplines_type = BSplines;
 
+	/**
+     * @brief The deriv dimension at the boundaries.
+     */
     using deriv_type = ddc::Deriv<tag_type>;
```

Comment:
Bad indentation.

Can you use the word `Deriv` in the description so the doc will (eventually) redirect to the docs of `Deriv`. Pauline had to update her code recently to take this into account and did not understand what this was supposed to represent.